### DISCUSSION TITLE
OSC2024 東京春（3/10 リアル展示）へのブース出展について

### DISCUSSION THREAD

**Discussion by potappo** [mozilla-japan/translation#737] [General]
前回 #711

OSC2024 東京春の日程は 2024年3月10日(日) 10:00～16:00 です。2/2 までブース出展を募集しています。
https://event.ospn.jp/osc2024-spring/
場所は去年の春と同じ、浅草駅最寄の東京都立産業貿易センター台東館となります。
私の予定は空いていますので、今回も出展申込を行います（1/20～1/21申込予定）。申込が受理されたら報告します。

会場に関することですが、原則去年と同じで、展示用にネットワークが使えるのは確定のようです。会場宛の宅配便は使えないので、展示物・配布物はすべて出展者が直接持ち込みになります。昼食は建物内に食堂があります。
出展を手伝っていただける方はこのディスカッションに連絡ください。時間調整のための調整さんを後日作成予定です。

**Comment by potappo** [comment]
さきほど出展申込をしました。

以下は一般来場者含めた参加登録ページです。今年は時間指定は特にないようです。
https://ospn.connpass.com/event/304655/

**Reply by potappo** [reply]
こちらで報告し忘れていましたが、申込は受理されました。ブースのレイアウトは 2 月の翻訳ミートアップにまで決まっていれば、そこで報告します。

**Reply by potappo** [reply]
Slack にレイアウト案の PDF を投稿しました。入口から数えると 1 列目で、わかりやすい位置かと思います。
まだ調整できるので、何か意見がある人がいればお願いします。

**Comment by gurezo** [comment]
@potappo @hmatrjp 
以前、OSC運営事務局にグッズ提供の打診をして回答を共有していませんでした。
スレッドの内容は下図のとおりです。

![スクリーンショット 2024-02-09 23 25 18](https://github.com/mozilla-japan/translation/assets/7631567/8ff8c702-cf76-4213-a473-6023ddb56d32)

上図の通り、現在プレゼント企画をしていないので受付予定はないようです。
当日にグッズを搬入し「ご自由にお持ち下さい」コーナーに配置（提供）する場合は、問題ないようです。
コミュニティとして、どう対応するかご意見頂けたら幸いです。

**Reply by hmatrjp** [reply]
びぎねっとさんへの問い合わせと情報共有ありがとうございます。 @gurezo 
「ご自由にお持ちください」コーナーに置くなら、水転写式のデカールとかでしょうか。（ある程度、在庫数があって、軽い。追加説明は必要かも。）

私が想定していたのは、金属製のタブレットフォルダーとかマグカップ（ビンゴの景品になるような品）でした。これらはやめたほうが良さそうですね。
- 重いので、当日搬入だと、厳しいかもしれない
- 人が介在しない受け渡し方に（心情的な）抵抗がある

**Reply by hmatrjp** [reply]
ほかに、これならいいんじゃないかとか、ご意見あればお願いします。　在庫表へのリンクはこちらです。　> みなさま
https://docs.google.com/spreadsheets/d/1N73mciqvlQbuC4457OMteS-TXDldj75XybDKfK8CToI/edit#gid=345022493

**Reply by gurezo** [reply]
@hmatrjp cc @potappo 
シートの J, K 列に「ご自由に」と「ブース」のグッズに「◯」を付けてみました。
皆さんのご意見を聞きたいです。

**Reply by potappo** [reply]
@gurezo @hmatrjp 
グッズについてありがとうございます。

確かに、重い物は今回の当日搬入だと難しいですね。

ぐれさんのグッズの提案を見ましたが、帽子とポーチも重くはないものの、マグカップに近いレアリティを感じるので、ブース訪問者の側としても手に取りにくいかも？という気がしています。

ブースで配れそうなのは、「スクリーン拭き」はどうでしょうか。説明が英語表記なのでご自由にの場所に置くのは多少気になるかもですが、ブースなら良いと思います。

他の提案については特に異論はないです。

**Reply by gurezo** [reply]
@potappo 

> 帽子とポーチも重くはないものの、マグカップに近いレアリティを感じるので、ブース訪問者の側としても手に取りにくいかも？という気がしています。

なるほどです。ポーチは外しました。

**Reply by hmatrjp** [reply]
巻取り式ワイヤーも、ブースでなら配れるかもしれません。ぱっと見では、用途と使い方がわからないかも。

**Reply by potappo** [reply]
今日のミートアップで話したことのメモ：
- グッズは 2月中に potappo 宛に送ってもらう（会場へは私がバッグに入れて持参します）。

**Reply by hmatrjp** [reply]
今日、グッズを @potappo さん宛に発送しました。日付時間無指定（窓口で聞いたところでは明日午前着予定）です。
荷姿は、26cm × 19cm × 12 cm のダンボール箱です。
数量と品は、 [在庫表](https://docs.google.com/spreadsheets/d/1N73mciqvlQbuC4457OMteS-TXDldj75XybDKfK8CToI/edit#gid=345022493) の L列に追記しました。
potappo さんの在庫(E列)と合わせれば、昨年のOSC春の配布実績と同等に近くなると思います。

**Reply by potappo** [reply]
@hmatrjp さん、送付ありがとうございます。
今日、自宅に届いて、仕事後に中身が問題ないことを確認しました。前日までダンボール箱で保管し、前日にカバンへと移して当日に会場へ持参します。今週末、収納状態や重さ確認のために、試しに、カバンへ入れてみる予定です。

**Reply by hmatrjp** [reply]
到着の報を聞いたあとに書き換えようと思って忘れていましたが、
[在庫表](https://docs.google.com/spreadsheets/d/1N73mciqvlQbuC4457OMteS-TXDldj75XybDKfK8CToI/edit#gid=345022493)のD列（濱崎在庫）から、出庫した分を引きました。

**Reply by potappo** [reply]
在庫表を更新しました。「在庫 potappo」列と「OSC Tokyo Spring 2024開催後在庫 potappo」列が等しい数になるようにしています。
今回の配布数は「OSC Tokyo Spring 2024」列を参照してください。

以下、全部ではないですが、各グッズの配布結果です。

会場へは直接持ち込みなので、ノート無地とボールペン付きノートを小さなキャリーバッグに入れて、カバン 2 つ体制で挑みましたが、ノート無地がほぼ配布できて良かったです。学生さん含めて、幅広い方が来場されてもらってくれました。

Firefox デカールは一部を「ご自由にお持ちください」コーナーに置いていましたが、それらもコーナーに残っていなかったので、全部配布できました。

moz://la ステッカーはスペースの都合もあり、はさみで分割して配っていました。残り1枚となっているのも、切り取ったものの1枚です。

その他シールは、出せなかったものもあり、まだ在庫があります（出さなかったのはBUDDY QUEST）。

ミニポケットはあまりもらい手がなく、在庫がそれなりに残っています。

**Reply by hmatrjp** [reply]
在庫表の更新ありがとうございます。セルE1にも、日付を入れました。 @potappo 
セル背景を、D列と同じように、残数0 のところを灰色に、ほかは白にしたいのですが、現在のカラーリングになにか意図はありますか？ 

moz://a ステッカーの切り分け配布は、配れる数が増えて、いいアイディアだと思います。

Buddy Quest のステッカーは、2010という年が入っているので、コレクターズアイテムのような(?)位置づけになってしまい、一般受けはしないかもしれませんね。

**Reply by potappo** [reply]
私のカラーリングに独自の意図はないので統一してもらってかまいません。

**Reply by hmatrjp** [reply]
ありがとうございます。統一しました。

**Comment by potappo** [comment]
調整さんを作成しました。当日ブース参加する方は入力をお願いします。

https://chouseisan.com/s?h=34017e6ef64d4b578b2e1b9f2af9a846

**Reply by gurezo** [reply]
入力 ✅

**Comment by potappo** [comment]
山本さんが作成中の動画についてのメモをここに書き残しておきます。

- 今回の OSC までに間に合ったら、展示用 PC のローカルにダウンロードして、ブースで観られるようにする。
- 恒久的には Mozilla Japan のチャンネルで公開する。動画が出来てから管理者を探して打診。 https://www.youtube.com/@MozillaJP

**Comment by potappo** [comment]
今回も WebDINO の井上さんからチラシ50部印刷と、展示用ノートPC（前回と同じもの）をお借りしました。
既に私の手元に到着済です。セットアップは今週のどこかか、3/3(日）までに実施します（最終的なアップデートは9日に実施）。

**Reply by potappo** [reply]
ほぼ前回同様のセットアップはしました。以下の作業が残です。
- [x] 動画のHTMLファイル作成（videoタグだけで良さそうだが、必要なら video.js を入れるかも）
- [x] 翻訳ガイドをローカルで表示できるようにする（開始と同時にネットワーク提供開始されなかった場合の対策）
- [x] PowerToys を入れて Awake を設定できるようにする（PC スタンバイ防止）

**Comment by potappo** [comment]
当日は特に大きなトラブルもなく無事に終了しました。みなさまありがとうございました。

チラシは 50 枚あったのが 12 枚残っているので、38 枚配布したことになります。ブース来場者数はカウントしていませんでしたが、チラシ配布数がブース来場者数とほぼ等しいはずです。OSC 全体の来場者数は閉会アナウンス時の速報値で 530 人とのことでした。

グッズ残数はグッズのスレッドで後ほど報告します。

**Comment by gurezo** [comment]
イベントが終了したので、close します。
